build/containers/Dockerfile.video-streaming-app

```diff
@@ -4,14 +4,14 @@ FROM ${PLATFORM}/debian:bullseye-slim
 # Avoid tzdata prompt 
 ARG DEBIAN_FRONTEND=noninteractive
 
+WORKDIR /app
+COPY ./samples/apps/video-streaming-app .
+
 RUN echo "Creating container based on ${PLATFORM}/debian:bullseye-slim" && \
     apt-get update && \
     apt-get install -y protobuf-compiler libprotoc-dev python3-pip python3-grpcio python3-kubernetes && \
```

Comment:
maybe remove `python3-grpcio` and `python3-kubernetes` from here, as they'll get installed via pip later